publish.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+ORG=elc9aya2ls612j
```

Comment:
I assume there is a good reason but why is the org so weird ?

```suggestion
ORG=adobe
```

Reply:
I've been trying some stuff with IDNs in Helix and still had the org hanging around